oxidation/libparsec/crates/client_connection/src/config.rs

```diff
@@ -0,0 +1,24 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
+
+#[cfg(not(target_arch = "wasm32"))]
```

Comment:
I'd rather avoid cluttering the code with `cfg(target_arch = ...)`, instead:

- `platform_http_proxy` is always built (so no conditional `cfg(target_ach = ...)`)
- `platform_http_proxy` only exposes a single function that takes the url to connect to and return a [reqwest Proxy object](https://docs.rs/reqwest/latest/reqwest/struct.Proxy.html)
- `platform_http_proxy`'s web implementation always return `NoProxy`

Reply:
For the body of the PR

> Note that proxy configuration isn't possible on wasm.
> Reqwest does not expose Proxy when compile on wasm

Reply:
ok, then can the function takes the url and a [`ClientBuilder`](https://docs.rs/reqwest/latest/reqwest/struct.ClientBuilder.html), then add configuration (only if not on web ^^) to the `ClientBuilder` and return it  ?

Reply:
That would be possible, but I would like to not pass and url and basically do the job of the proxy